### ISSUE TITLE
Include more FIPS tests into GH actions

### ISSUE THREAD

**Issue by mposolda** [keycloak/keycloak#14970]
We added some initial GH actions support in #14886 , but we may need to add more groups and more tests to GH actions.

We should add especially failing tests after #14886 was done (So tests for SAML, X509, Kerberos, KcRegTest etc). See #14964 - #14969 for the details about failing tests.

**Comment by mposolda** [comment]
This was already done. New tests are added continuously to FIPS testsuite when particular related issues are fixed (For example tests related to keystores were added once keystore was fixed).
